tests/benchdnn/pool/ref_pool.cpp

```diff
@@ -20,6 +21,20 @@
 
 namespace pool {
 
+float get_initial_min_value(const prb_t *prb) {
+#if DNNL_AARCH64_USE_ACL
+    // Ref: https://github.com/oneapi-src/oneDNN/issues/1205
+    // Changes required to support ACL which use std::numeric_limits<float>::infinity()
+    // for the minimum value of type float.
+    if (prb->cfg[DST].dt == dnnl_f32 && prb->is_acl == true)
+        return -std::numeric_limits<float>::infinity();
```

Comment:
I'm looking into TensorFlow and they use lowest value as oneDNN does: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/kernels/maxpooling_op.cc#L130
I'm not sure it's going to fly with a different default value. TF tests with kernels in padded area will fail for sure.
I would definitely double check with maintainers that this is an expected behavior once ACL pooling is triggered.

Reply:
We've decided it's best to skip all of these cases (where the -infinity issue arises) now